aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -913,7 +915,7 @@ Tensor _NestedTensor_GeneralizedBMM(const Tensor& self, const Tensor& mat2) {
   // create a contiguous output
   std::vector<int64_t> batch_sizes;
   Tensor output;
-  std::tie(batch_sizes, output) = _NestedTensor_GeneralizedBMM_BatchSizes_OutputMemory(
+  std::tie(batch_sizes, output) = matmul_nested_bmm_BatchSizes_OutputMemory(
```

Comment:
The mix of two naming conventions in a single name is weird lol

Reply:
right :face_with_spiral_eyes: things get confusing when the return value has a 2-word name (batch_size, output_memory)